test/unit_tests/graphql/utils/test_override_class.py

```diff
@@ -0,0 +1,293 @@
+import json
+from http import HTTPStatus
+from typing import Union
+from uuid import UUID
+
+import pytest
+import strawberry
+
+from orchestrator.graphql.schemas.customer import CustomerType
+from orchestrator.graphql.utils.override_class import override_class
+from test.unit_tests.config import GRAPHQL_ENDPOINT, GRAPHQL_HEADERS
+
+
+@pytest.fixture
+def override_class_app_graphql(
+    fastapi_app_graphql,
+    test_client,
+    sub_one_subscription_1,
+):
+    def customer_id_override(self) -> str:
+        return "overriden"
+
+    customer_id_override_field = strawberry.field(
+        name="customerId", resolver=customer_id_override, description="Returns customer_id"
+    )
+    customer_id_override_field.name = "customer_id"
+
+    def new(self) -> UUID:
+        return self.customer_id
+
+    new_field = strawberry.field(name="newField", resolver=new, description="Returns new_field")
+    new_field.name = "new_field"
```

Comment:
Can new_field just be constructed with `name="new_field"` directly?

Reply:
tested again, the `strawberry.field(name="newField")` doesn't seem necessary.